### DISCUSSION TITLE
Windows 11 UTM on M1 pro "Let's connect you to a network"

### DISCUSSION THREAD

**Discussion by BrendonHenshaw** [utmapp/UTM#5710] [Windows]
### Discussed in https://github.com/utmapp/UTM/discussions/3870

<div type='discussions-op-text'>

<sup>Originally posted by **Ramez-Dous** April  4, 2022</sup>
Hello,

I'm stuck with this error when installing Windows 11 on my MacBook Pro 16 "M1 Pro". 

"
Let's connect you to a network
You'll need an internet connection to
continue setting up your device. Once
connected, you'll get the latest features
and security updates.

Having trouble getting connected?
For troubleshooting tips, use another
device and visit aka.ms/networksetup 
"

Tried deleting and creating the VM many times. Also, tried Shared Network, Bridged, and Emulated VLAN. Tried many times with different Emulated Network Card. Also, Tried on WiFi and a USB to Ethernet. Nothing seems to work.

Any ideas?!

Thank you!</div>

**Comment by BrendonHenshaw** [comment]
Windows 11 UTM om M1 Air "Wifi Settings"

Hi everyone
Q: The Wifi is working great at Home, but when I go to the cafe I cannot find any settings to log into the Wifi at my church. How do I log into the Wifi at my church?

Note: the reason why I need to do this is because I want to use this MacBook to livestream our zoom meetings through OBS Studio to YouTube live, this is how we plan to stream live to both FaceBook and YouTube for free. Our old laptop can only just do the zoom meetings and is not powerful enough to run OBS Studio at the same time. I tried to live Stream on macOS and it couldn't pick up the zoom meetings video, plus I had an echo on the sound, so I installed blackhole 2ch and then the audio stopped altogether and I can't get sound back into OBS studio after so many attempts, I had also tried to do the same on a MacBook Pro 2014 that I used to have and that failed just the same, but I set up Windows 10 on it and it worked perfectly. I wish I still had the MacBook Pro 2014 as at it wasn't to bad to set up, so far I'm wasting hours and hours with this MacBook Air M1 and I might have to sell it and buy a second hand MacBook Pro 2014. Is it just meow are the new Apple silicon MacBook and new versions of MacOs just unable to do things like Windows 10 or 11 or OBS Studio properly? Window 10 and Windows 11 just work so good and work well on old MacBook Pro's

**Comment by SamiAdamMoughli** [comment]
Did you get any answer yet?
